dmn-kotlin-dsl/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
target/ is in the global one. Is it not good enough there?

Reply:
ok